apps/omg_watcher_info/lib/omg_watcher_info/db/eth_event.ex

```diff
@@ -83,6 +84,7 @@ defmodule OMG.WatcherInfo.DB.EthEvent do
               blknum: blknum,
               txindex: 0,
               oindex: 0,
+              otype: WireFormatTypes.output_type_for(:output_payment_v1),
```

Comment:
will otype come from the data received in the ethevent ethereum logs? ie: the contracts send this info already or have been updated to do so?

Reply:
The otype is not directly included in the ethereum event log, as you can see here: https://github.com/omisego/plasma-contracts/blob/2c152a24ee22e577ca182d3734ae422b545a2ebc/plasma_framework/contracts/src/vaults/EthVault.sol

As we only have payment type for now that can be deposited, I followed the code that was already present here: https://github.com/omisego/elixir-omg/blob/master/apps/omg/lib/omg/state/core.ex#L486 where we assume that a deposit output is for a payment.